composer.json

```diff
@@ -250,9 +250,9 @@
             "@monorepo-tools"
         ],
         "cs-fixer": [
-            "tools/ecs/vendor/bin/ecs check *-bundle/bin *-bundle/src *-bundle/tests --config tools/ecs/vendor/contao/easy-coding-standard/config/default.yaml --fix --ansi",
-            "tools/ecs/vendor/bin/ecs check *-bundle/src/Resources/contao --config tools/ecs/vendor/contao/easy-coding-standard/config/legacy.yaml --fix --ansi",
-            "tools/ecs/vendor/bin/ecs check *-bundle/src/Resources/contao/templates --config tools/ecs/vendor/contao/easy-coding-standard/config/template.yaml --fix --ansi"
+            "tools/ecs/vendor/bin/ecs check *-bundle/bin *-bundle/src *-bundle/tests --config tools/ecs/vendor/contao/easy-coding-standard/config/default.php --fix --ansi",
+            "tools/ecs/vendor/bin/ecs check *-bundle/src/Resources/contao --config tools/ecs/vendor/contao/easy-coding-standard/config/legacy.php --fix --ansi",
+            "tools/ecs/vendor/bin/ecs check *-bundle/src/Resources/contao/templates --config tools/ecs/vendor/contao/easy-coding-standard/config/template.php --fix --ansi"
```

Comment:
I guess https://github.com/contao/contao/blob/2cb7e68390f3c29004fca9f5616ba13647806675/.github/workflows/ci.yml#L66-L68 needs to be updatet too?

Reply:
Yes. 🤦🏻‍♂️